### PR TITLE
Add access controll

### PR DESCRIPTION
**Pull request by RobertAlexandroaie**

### CODE REVIEW

File: frolicaide/apps/athlete/fixtures/superuser.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      "last_name": "",
+      "email": "",
+      "is_staff": true,
+      "is_active": true,
+      "date_joined": "2020-06-20T15:11:57.658Z",
+      "groups": [],
+      "user_permissions": []
+    }
+  }
+]
```
Comment (cbicsabi) [inline frolicaide/apps/athlete/fixtures/superuser.json:19]: Should be kept only in 1Password due to password exposure

File: frolicaide/apps/athlete/forms.py [hunk 2]
```diff
@@ -9,8 +9,6 @@ class DateInput(forms.DateInput):
 
 
 class AthleteForm(forms.ModelForm):
-    club = forms.CharField(initial='')
-    federation = forms.CharField(initial='')
```
Comment (cbicsabi) [inline frolicaide/apps/athlete/forms.py:13]: clean_club and clean_federation must be also deleted

File: frolicaide/settings/production.py [hunk 3]
```diff
@@ -159,3 +159,6 @@
 
 # Activate Django-Heroku.
 django_heroku.settings(locals(), test_runner=False, staticfiles=False, allowed_hosts=False)
+
+# Redirect to home URL after login (Default redirects to /accounts/profile/)
+LOGIN_REDIRECT_URL = '/'
```
Comment (cbicsabi) [inline frolicaide/settings/production.py:164]: Pune sus

File: frolicaide/apps/athlete/fixtures/superuser.json (hunk 1 above)
Comment (RobertAlexandroaie) [reply frolicaide/apps/athlete/fixtures/superuser.json]: addressed

File: frolicaide/settings/production.py (hunk 3 above)
Comment (RobertAlexandroaie) [reply frolicaide/settings/production.py:164]: las' asa :))

**Pull request merged**